### PR TITLE
op-node: Fix OPB-07

### PR DESCRIPTION
**Pull request by mslipper**
Fixes Sigma Prime audit issue OPB-07: Private Key Stored Without Encryption. This PR requires reading the sequencer private key from an environment variable rather than a file on disk.

Fixes ENG-2616

### CODE REVIEW

**Review by trianglesphere — APPROVED** [review]

File: ops-bedrock/docker-compose.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ services:
       --sequencer.enabled
       --sequencer.l1-confs=0
       --verifier.l1-confs=0
-      --p2p.sequencer.key=/config/p2p-sequencer-key.txt
```
Comment (tynes) [inline ops-bedrock/docker-compose.yml:54]: You can now delete the file that holds this key as well right?

**Comment by tynes** [comment]
I'm guessing that the hive dockerfiles are now breaking..

**Comment by mslipper** [comment]
Correct. Will fix that in a separate PR.

**Comment by mslipper** [comment]
https://github.com/ethereum-optimism/hive/pull/44 fixes this in Hive.

**Review by protolambda — APPROVED** [review]

**Pull request merged**